Inc/info.h

```diff
@@ -0,0 +1,103 @@
+/**
+@file vport.h
+@brief Generates the device info blob and writes it over serial.
+*/
+
+#pragma once
```

Comment:
please encapsulate header file in #ifndef / #endif in addition to #pragma once to ensure maximum compatibility